test/meta/src/RuntimeTypes.c

```diff
@@ -1580,47 +1709,45 @@ void RuntimeTypes_nested_struct_with_strings(void) {
 
     /* Test constructor: */
     ecs_entity_t e = ecs_new(world);
-    {
-        NestedStructWithStrings *ptr =
-            ecs_ensure_id(world, e, nested_struct_with_strings);
-        test_memory_zero(ptr, sizeof(NestedStructWithStrings));
-        ptr->a.a = ecs_os_strdup("String100");
-        ptr->a.b = 101;
-        ptr->a.c = ecs_os_strdup("String102");
-        ptr->b = 103;
-        ptr->c.a = ecs_os_strdup("String104");
-        ptr->c.b = 105;
-        ptr->c.c = ecs_os_strdup("String106");
-    }
+    NestedStructWithStrings *ptr1 =
+        ecs_ensure_id(world, e, nested_struct_with_strings);
+    test_memory_zero(ptr1, sizeof(NestedStructWithStrings));
+    ptr1->a.a = ecs_os_strdup("String100");
+    ptr1->a.b = 101;
+    ptr1->a.c = ecs_os_strdup("String102");
+    ptr1->b = 103;
+    ptr1->c.a = ecs_os_strdup("String104");
+    ptr1->c.b = 105;
+    ptr1->c.c = ecs_os_strdup("String106");
 
     /* Test copying: */
     ecs_entity_t instance = ecs_clone(world, 0, e, true);
-    {
-        const NestedStructWithStrings *ptr =
-            ecs_get_id(world, instance, nested_struct_with_strings);
-        test_str("String100", ptr->a.a);
-        test_int(101, ptr->a.b);
-        test_str("String102", ptr->a.c);
-        test_int(103, ptr->b);
-        test_str("String104", ptr->c.a);
-        test_int(105, ptr->c.b);
-        test_str("String106", ptr->c.c);
-    }
+    const NestedStructWithStrings *ptr2 =
```

Comment:
Not going to fail the PR on this, but the `{ }` where there so you don't have to give the variables different names, which makes code easier to extend/copy paste. For future ref please don't remove `{ }`